### ISSUE TITLE
Support image transformation in XShards

### ISSUE THREAD

**Issue by songhappy** [intel-analytics/BigDL#6825]
This issue is to track image support on Orca Xshards.

**use case example to be adapted into tutorials of Xshards.**
1. image classification
    https://www.kaggle.com/code/ruchibahl18/cats-vs-dogs-basic-cnn-tutorial
2. basics 
    https://www.tensorflow.org/tutorials/load_data/images
3. face mask detection 4. 
    https://www.kaggle.com/code/sandhyakrishnan02/face-mask-detection-using-pytorch

**common libraries transformations and references. **
1. opencv is chosen since it is faster based on bench mark(https://www.kaggle.com/code/vfdev5/pil-vs-opencv) 
2. transformation basics
    Basics: crop, resize, normalize
    Augmentation: flip, rotate, dilate, add_light, gaussian blur
    https://www.kaggle.com/code/bulentsiyah/learn-opencv-by-examples-with-python
    https://www.kaggle.com/code/parulpandey/overview-of-popular-image-augmentation-packages
    https://www.kaggle.com/code/khotijahs1/cv-image-preprocessing
    https://www.analyticsvidhya.com/blog/2021/05/image-processing-using-opencv-with-practical-examples/

**dataset**
CIFAR: https://www.cs.toronto.edu/~kriz/cifar.html
Dogs-vs-Cats: https://www.kaggle.com/code/ruchibahl18/cats-vs-dogs-basic-cnn-tutorial/data?select=test1.zip
Face-mask-data: https://www.kaggle.com/code/sandhyakrishnan02/face-mask-detection-using-pytorch/data
